src/components/Navigation/Footer.vue

```diff
@@ -1,8 +1,13 @@
 <template>
   <v-row>
-    <v-col cols="3">
+    <v-col
+      cols="3"
```

Comment:
We have want a column class to 12 or 6 for very small screen but it's just a suggestion.